### PR TITLE
Add support for Crema Grande

### PR DESCRIPTION
**Pull request by Ki-qu**
Commercial name of product: Crema Grande
Android Version: 4.4.2
Light: Tolino Ntx
Display: Freescale/NTX

Manufacturer: iwg
Brand: crema
Model: crema-0710c
Device: ntx_6sl
Product: ntx_6sl
Hardware: e60qn0
Platform: imx6

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/android-luajit-launcher/495)

### CODE REVIEW

**Comment by Ki-qu** [comment]
I found better solution that works with this device without artificial delay by reverse engineering stock apps: by writing a value to `/sys/class/graphics/fb0/full_refresh_request`.

- `1` will refresh the screen after screen is updated (This is what I'd like to use)
- `2` will immediately refresh the screen

This might be only relevant to my device. Can I implement with this method?

**Comment by Frenzie** [comment]
Related, does that require root?

**Comment by Ki-qu** [comment]
No. It doesn't require root.

File: app/src/main/java/org/koreader/launcher/device/epd/TolinoAltEPDController.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/* Tested on Crema Grande */
```
Comment (Frenzie) [inline app/src/main/java/org/koreader/launcher/device/epd/TolinoAltEPDController.kt:1]: What else would it be tested on. ;-)

```suggestion
```

**Review by Frenzie — APPROVED** [review]
lgtm, pinging @pazos

**Comment by Ki-qu** [comment]
Sorry. Upon further testing I found that my hacky approach works rather inconsistently. I changed it back to NookEPDController.

**Pull request merged**